libr/core/dmh_glibc.inc.c

```diff
@@ -108,13 +317,9 @@ static bool GH(is_tcache)(RCore *core) {
 	RListIter *iter;
 	r_debug_map_sync (core->dbg);
 	r_list_foreach (core->dbg->maps, iter, map) {
-		// In case the binary is named *libc-* or *libc.*
+		// In case the binary is named *libc*
 		if (strncmp (map->name, core->bin->file, strlen (map->name)) != 0) {
-			char *fp = strstr (map->name, "libc-");
-			if (fp) {
-				break;
-			}
-			fp = strstr (map->name, "libc.");
+			char *fp = strstr (map->name, "libc");
 			if (fp) {
 				v = r_num_get_float (core->num, fp + 5);
```

Comment:
"libc" is 4 letters, fp+5 can cause an oobread.

Reply:
this is all removed now